src/components/Table/Table.tsx

```diff
@@ -70,6 +70,40 @@ export const HeaderCell = styled(TH)`
   `}
 `;
 
+export const Footer = styled(TR)`
+  ${({ columnGap, columnWidths }: RowProps) => {
+    const { colors } = useTheme();
+    return `
+      display: grid;
+      grid-template-columns: ${columnWidths};
+      padding: 0rem 2rem;
+      column-gap: ${columnGap};
+      user-select: none;
+
+      background-color: ${colors.grayXlight};
+      color: black;
```

Comment:
I think we have a color enum for black - if header uses the CSS color like this we should update that as well